node/bft/src/primary.rs

```diff
@@ -110,6 +112,7 @@ impl<N: Network> Primary<N> {
 
     /// Initializes a new primary instance.
     pub fn new(
+        block_sync: Arc<BlockSync<N>>,
```

Comment:
nit: I would move `block_sync` to be after `ledger` (so that the related items are close to one another, and the order of arguments maintains a relative dependency tree of hierarchy/layers)

Reply:
(with an IDE, you should be able to reorder these arguments and all declarations/instantiations of this method with 1 refactor button-click)